Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -39,14 +39,20 @@ function Get-AzureUri
 function Get-ProxyUri
```

Comment:
I believe the newer version of DeployRG has moved to Node based and is no longer taking dependency on PS modules. Given this can you revert the patch version update done on that task.